rubicon/objc/__init__.py

```diff
@@ -3,6 +3,7 @@
 from .objc import (
     objc, send_message, send_super,
     get_selector,
+    SEL, objc_id, Class, IMP, Method, Ivar, objc_property_t,
```

Comment:
Yes, `IMP` is not used much. It's mainly used in some of the function declarations (such as `class_addMethod` or `method_getImplementation`) as a placeholder type for any function pointer. Because of this, `IMP` needs to be publicly available, to allow others to call the functions that have an `IMP` in their signature.